neqo-client/src/main.rs

```diff
@@ -956,6 +957,10 @@ fn qlog_new(args: &Args, hostname: &str, cid: &ConnectionId) -> Res<NeqoQlog> {
 async fn main() -> Res<()> {
     init();
 
+    if std::env::var("RUST_LOG").is_err() {
+        eprintln!("Set RUST_LOG=<level> for logs.")
+    }
```

Comment:
A bit odd, but probably better than surprising folks that are used to logs appearing automatically. Happy to remove.

Reply:
Maybe add a `-v` command line parameter to increase the log level?